<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Anushas-MacBook-Pro.local" failures="4" tests="32" name="test_runner_team08.TestRunner" time="10.534" errors="0" timestamp="2024-03-31T03:32:01 EDT" skipped="0">
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.293"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.132"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.389">
    <failure message="expected:&lt;400&gt; but was:&lt;201&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<201>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:472)
at org.junit.Assert.assertEquals(Assert.java:456)
at utilities_team08.CommonValidation.statusValidations(CommonValidation.java:68)
at step_definition_team08.UserModuleStepDefinition.admin_receives_bad_request_status_with_message_and_boolean_success_details(UserModuleStepDefinition.java:192)
at ✽.Admin receives 400 Bad Request Status with message and boolean success details(file:///Users/anushakarthick/NumpyNinja/Anusha_Team8_APIArchitects_RestAssured/src/test/resources/features/UserModule1.feature:32)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.120"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.122"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="1.047"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.132"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.148"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.134"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.482">
    <failure message="expected:&lt;400&gt; but was:&lt;201&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<201>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:472)
at org.junit.Assert.assertEquals(Assert.java:456)
at utilities_team08.CommonValidation.statusValidations(CommonValidation.java:68)
at step_definition_team08.UserModuleStepDefinition.admin_receives_bad_request_status_with_message_and_boolean_success_details(UserModuleStepDefinition.java:192)
at ✽.Admin receives 400 Bad Request Status with message and boolean success details(file:///Users/anushakarthick/NumpyNinja/Anusha_Team8_APIArchitects_RestAssured/src/test/resources/features/UserModule1.feature:32)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.762"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.145"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.183"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.472">
    <failure message="expected:&lt;400&gt; but was:&lt;201&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<201>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:472)
at org.junit.Assert.assertEquals(Assert.java:456)
at utilities_team08.CommonValidation.statusValidations(CommonValidation.java:68)
at step_definition_team08.UserModuleStepDefinition.admin_receives_bad_request_status_with_message_and_boolean_success_details(UserModuleStepDefinition.java:192)
at ✽.Admin receives 400 Bad Request Status with message and boolean success details(file:///Users/anushakarthick/NumpyNinja/Anusha_Team8_APIArchitects_RestAssured/src/test/resources/features/UserModule1.feature:49)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.774"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.119"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.122"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.123"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.119"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.143"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="1.296"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.153"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.875"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.140"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.231"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.138"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.385">
    <failure message="expected:&lt;400&gt; but was:&lt;201&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<201>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:472)
at org.junit.Assert.assertEquals(Assert.java:456)
at utilities_team08.CommonValidation.statusValidations(CommonValidation.java:68)
at step_definition_team08.UserModuleStepDefinition.admin_receives_bad_request_status_with_message_and_boolean_success_details(UserModuleStepDefinition.java:192)
at ✽.Admin receives 400 Bad Request Status with message and boolean success details(file:///Users/anushakarthick/NumpyNinja/Anusha_Team8_APIArchitects_RestAssured/src/test/resources/features/UserModule1.feature:49)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.501"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.124"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.167"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.143"/>
  <testcase classname="test_runner_team08.TestRunner" name="runScenario" time="0.420"/>
</testsuite> <!-- test_runner_team08.TestRunner -->
